docs/reference/api.rst

```diff
@@ -366,6 +371,22 @@ Rest API
 
 The rest API is also available as a `swagger spec <openapi.html#http://>`_
 
+You can interact with these API endpoints using the :py:class:`inmanta.protocol.endpoints.Client` python client:
+
+.. code-block:: python
+
+    client = inmanta.protocol.endpoints.Client(name="client_name", timeout=120)
+
+    result = await client.environment_list(details=True)
```

Comment:
The issue here is: how do people find the method name... 

it is somewhere in the schema , if you look at the url, it is the last part of the anchor 
But we don't show it.  I think it is worth seeing if we can reconfigure the way we generate the docs to show the method name

https://sphinxcontrib-redoc.readthedocs.io/en/stable/
https://github.com/Redocly/redoc/issues/536


<img width="1742" height="515" alt="image" src="https://github.com/user-attachments/assets/f1cefde3-9c28-4d10-ab01-8cd23e05c374" />